src/corehost/build.sh

```diff
@@ -184,6 +184,13 @@ else
     exit 1
 fi
 
+if [ -z $CONFIGURATION ]; then
+    export CONFIGURATION=Debug
+fi
+echo "CONFIGURATION=$CONFIGURATION"
+
+__cmake_defines="-DCMAKE_BUILD_TYPE=${CONFIGURATION} ${__cmake_defines}"
```

Comment:
Now configruation "Release" and "Debug" is passed to corehost correctly. I made a PR to add this feature in arm CI. https://github.com/dotnet/core-setup/pull/1641